apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/model/CreateRuleRequest.java

```diff
@@ -0,0 +1,28 @@
+package gov.cdc.nbs.questionbank.model;
+
+public sealed interface CreateRuleRequest {
+   String ruleFunction();
+   String ruleDescription();
+
+   String anySourceValue();
+   String comparator();
+
+   String source();
+   String sourceValue();
+   String targetType();
+
+   String targetValue();
+
+    record ruleRequest(
+            String ruleFunction,
+            String ruleDescription,
+            String source,
+            String anySourceValue,
+            String comparator,
+            String sourceValue,
+            String targetType,
+            String targetValue) implements CreateRuleRequest{ }
+
+
+}
```

Comment:
May not need the enclosing interface since I don't think there will be more than one implementing record. 

Could just have a `CreateRuleRequest` record with all of the fields.
Correct me if I'm wrong or missing something.